Core/Items/ItemTooltips.cs

```diff
@@ -104,6 +110,30 @@ public override bool PreDrawTooltipLine(Item item, DrawableTooltipLine line, ref
 			return true;
 		}
 
+		if (line.Name.StartsWith("Separator"))
```

Comment:
This may result in conflicting mod behavior as the the name does not seem very unique. You could do the following and make AddSeparator always use PoTMod.Instance for the Mod field.
```suggestion
		if (line.Name.StartsWith("Separator") && line.Mod == Mod.Name)
```

Reply:
The Separator line always runs after `if (line.Mod != Mod.Name)`, so that's redundant.